.github/workflows/release.yaml

```diff
@@ -112,6 +112,11 @@ jobs:
               chainloop attestation add --name ${BINARY_NAME} --value ${BINARY_PATH} || true
             done
 
+      - name: Add Attestation Artifacts (source code)
```

Comment:
related to a comment from @danlishka this morning. Is there any way to find this source code tgz from the github release itself instead of crafting our own?

![image](https://github.com/chainloop-dev/chainloop/assets/24523/ffb8d174-33f3-4019-8dd8-b68016ed0967)

Reply:
Hmm I'll try to read about it and know where is that taken from. Maybe that's generated after the release process or in a different step that we don't have access.